scripts/cluster/setup_worker_kubelet.go

```diff
@@ -51,7 +51,7 @@ func CreateWorkerKubeletService(criSock string) error {
 	}
 	bashCmd := "sudo sh -c 'cat <<EOF > /etc/systemd/system/kubelet.service.d/0-containerd.conf\n" +
 		"[Service]\n" +
-		`Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix://%s"` +
+		`Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --v=5 --kube-api-qps=50000 --kube-api-burst=100000 --runtime-request-timeout=15m --container-runtime-endpoint=unix://%s"` +
```

Comment:
@cvetkovic Do you think these API qps changes are good-to-have everytime running vHive? Can they be applied on demand based on the needed values for experiment?

Reply:
These values were experiment-specific.

Reply:
So should we remove them then?

Reply:
The default values are 5 and 10, I think. I would put 50 and 100 and that would be it.

Reply:
@wanghanchengchn please implement